Prog/Eviction Group/Pulse-housing-analysis.R

```diff
@@ -0,0 +1,141 @@
+           moved = case_when(MOVED == 1 ~ "Moved from Pressure", 
+                             MOVED == 2 ~ "Did not move from Pressure",
+                             MOVED == -99 | MOVED == -88 ~ "Did not report"),
+           income = case_when(INCOME == 1 ~ "$25,000 below",  
+                            INCOME == 2 | INCOME == 3 ~ "$25,000-$49,999",
+                            INCOME == 4 ~ "$50,000-$74,999",
+                            INCOME == 5 ~ "$75,000-$99,999",
+                            INCOME == 6 | INCOME == 7 | INCOME == 8 ~ "$100,000 above",
+                            INCOME == -99 | INCOME == -88 ~ "Not reported"),
+           rent_behind = case_when(RENTCUR == 1 ~ "Not behind on rent",
+                                   RENTCUR == 2 ~ "Behind on rent",
+                                   RENTCUR == -99 | RENTCUR == -88 ~ "Did not report"),
+           eviction_two_months = case_when(EVICT == 1 ~ "very likely", 
+                                           EVICT == 2 ~ "somewhat likely", 
+                                           EVICT == 3 ~ "not very likely", 
+                                           EVICT == 4 ~ "not likely at all",
+                                           EVICT == -99 | EVICT == -88 ~ "Did not report"),
+           inc_cat=case_when(INCOME == 1 | INCOME == 2 | INCOME == 3 ~ "below 40 AMI", #0-$49,999; #based on HUD 2024 incomes limits for income & household size 
+                             INCOME == 4 & THHLD_NUMPER >= 2 ~ "below 40 AMI", #50,000-75,000 & 2 person household or more
+                             INCOME == 5 & THHLD_NUMPER >= 7 ~ "below 40 AMI", #75-100k & 7 person household or more
+                             INCOME == -99 | INCOME == -88 ~ "Not reported",
+                             TRUE ~ "above 40 AMI")) %>%
+  select(SCRAM,HWEIGHT,PWEIGHT,WEEK,CYCLE,EST_ST,
+         pressured,moved,increase_rent,missed_rent,repairs_not_made,eviction_threatened,
+         locks_changed,nhbd_danger,other_pressure,
+         rent_behind,eviction_two_months,income,inc_cat,THHLD_NUMPER, INCOME) 
+
+all_PUF <- inner_join(clean_data, rep_weights, by = c("SCRAM", "WEEK", "CYCLE")) # need to join by week/cycle for replicate weights
+
+srvy_all <- 
+  as_survey_rep(
+    all_PUF,
+    repweights = dplyr::matches("HWEIGHT[0-9]+"),
+    weights = HWEIGHT,
+    type = "BRR",
+    mse = TRUE
+  )
+
+# 1) Households who think they are likely to face an eviction in the next two months
+eviction_behind_rent <-  srvy_all %>%
+  filter(inc_cat != "Not reported") %>% # remove unreported income levels
+  group_by(inc_cat, eviction_two_months) %>%
+  summarise(proportion = survey_mean()) %>%
+  filter(proportion_se < 0.05) %>% # taking out obs with large standard errors
+  select(-proportion_se) %>%
+  mutate_if(is.numeric, round, digits = 2) %>%
+  filter(eviction_two_months != "Did not report") %>%
+  pivot_wider(names_from = inc_cat, values_from = proportion) 
+
```

Comment:
If I'm understanding correctly, the values in the `above 40 AMI` and `below 40 AMI` columns are the proportions of observations for each value of `eviction_two_months` (not likely at all, not very likely, somewhat likely, and very likely) that fall in those `inc_cat `categories. Aren't those the only two categories of `inc_cat`, aside from "Not reported"? If so, would we expect these proportions to be larger?

Reply:
The share won't add up to 100% in this case because only respondents who answered that they're behind in rent saw the question, but for our publication we want the share of all renters, so keeping the denominator as all renters. 

Thank you for pointing this out! I needed to remove week 57 from total_pressure and total_moved because those questions were new to the survey starting in week 58. I filtered out the Not reported column in eviction_behind_rent but I'm going to remove that line to stay consistent.